File: DropZone/DropZone.Core/Models/Satchel/SatchelModel.cs
using System;

namespace DropZone.Core.Models
{
    public class SatchelModel
    {
        public long Id { get; set; }

        public double MainParachuteArea { get; set; }

        public double ReserveParachuteArea { get; set; }

        public DateTime? MaintenanceDate { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long Manufacturerid { get; set; }

        public long HashBlockId { get; set; }


        public ManufacturerModel Manufacturer { get; set; }

        public HashBlockModel HashBlock { get; set; }

        public ParachuteSystemModel ParachuteSystem { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/satchels/satchel.model.ts
import { ManufacturerModel } from "../manufacturers/manufacturer.model";
import { HashBlockModel } from "../hashBlocks/hashBlock.model";
import { ParachuteSystemModel } from "../parachuteSystems/parachuteSystem.model";

export class SatchelModel {
	public Id: number;
	public MainParachuteArea: number;
	public ReserveParachuteArea: number;
	public MaintenanceDate: Date;
	public ModifiedAt: Date;

	public ManufacturerId: number;
	public HashBlockId: number;
	
	public Manufacturer: ManufacturerModel;
	public HashBlock: HashBlockModel;
	public ParachuteSystem: ParachuteSystemModel;
};